checkstyle.xml

```diff
@@ -77,7 +77,7 @@
 
 		<module name="ImportOrder">
 			<property name="groups" value="java,javax,*,net.minecraft,net.fabricmc"/>
-			<property name="ordered" value="false"/><!-- the plugin orders alphabetically without considering separators.. -->
+			<property name="ordered" value="true"/>
```

Comment:
I couldnt see any issues with this, I imagine its an Eclipse thing, or has been fixed.